src/promptflow/tests/executor/e2etests/test_executor_execution_failures.py

```diff
@@ -26,6 +27,7 @@
 Traceback (most recent call last):
 in wrapped
     output = func(*args, **kwargs)
+             ^^^^^^^^^^^^^^^^^^^^^
```

Comment:
This is only worked in python 3.11

Reply:
the best way is not to check this line...

Reply:
check my code, I have removed this line if python < 3.11